src/server/pfs/fuse/filesystem_test.go

```diff
@@ -419,19 +497,26 @@ func testFuse(
 	require.NoError(t, err)
 	apiClient := pfsclient.NewAPIClient(clientConn)
 	mounter := fuse.NewMounter(localAddress, apiClient)
-
+	fmt.Printf("XXX created clientConn\n")
 	mountpoint := filepath.Join(tmp, "mnt")
 	require.NoError(t, os.Mkdir(mountpoint, 0700))
 	ready := make(chan bool)
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
+		fmt.Printf("XXX mounting\n")
 		require.NoError(t, mounter.Mount(mountpoint, nil, nil, ready))
 	}()
 
 	<-ready
+	fmt.Printf("XXX mounted\n")
+
 	defer func() {
+		fmt.Printf("XXX Trying to unmount\n")
 		_ = mounter.Unmount(mountpoint)
+		fmt.Printf("XXX Unmounted!\n")
 	}()
+	fmt.Printf("XXX running test callback\n")
 	test(apiClient, mountpoint)
+	fmt.Printf("XXX ran test callback\n")
```

Comment:
It was intentional, but happy to discuss. In the commit message for those changes I mention the reasons.

Basically for #346 there are several ways it seems to fail. Having those lines in the error output does differentiate a few of the error patterns. Since #346 is difficult to reproduce, and you only see that output when a fuse test fails (and you may not if you re-run it), i thought it might be helpful to leave them in so that as we see this issue again, we can add the log to the issue #346.